### ISSUE TITLE
Featured Tabs Block

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/student-loan-planner-theme#24]
- The tabs are missing the hover state. See design.
- The tab order doesn't match the design. Maybe this changed recently?
- The tab icons are blurry, use SVGs instead of PNGs
- The tabs are not accessible. This link was provided on the design. Please code to these standards.https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/ 
- Icons should be hidden from screen readers. aria-visialbe=false
- Add actual links to buttons instead of placeholders (#)
- Is the "2,400+ 5-star reviews" text supposed to be a link? It looks like a link. @amberhinds
- The tab focus outline is cut off at the top.
- Tabs are not responsive for tablet and mobile
- In the tab content the button doesn't match the design. See border-radius.
- In the block edit in preview mode the tabs don't work. I'm guessing the javascript is not being enqueued at the block level. See ACF docs.
- ACF fields
  - Change 'Add Row' to 'Add Tab'
  - Does the content need the full TinyMCE output and Add Media button?

Basecamp Ref: https://3.basecamp.com/3579237/buckets/32176113/card_tables/cards/6504307130
